PerfettoCds/Pipeline/Tables/PerfettoGenericEventTable.cs

```diff
@@ -38,8 +38,12 @@ public class PerfettoGenericEventTable
             new ColumnMetadata(new Guid("{d3bc5189-c9d1-4c14-9ce2-7bb4dc4d5ee7}"), "Name", "Name of the Perfetto event"),
             new UIHints { Width = 210 });
 
-        private static readonly ColumnConfiguration TimestampColumn = new ColumnConfiguration(
-            new ColumnMetadata(new Guid("{d458382b-1320-45c6-ba86-885da9dae71d}"), "Timestamp", "Android timestamp for the event"),
+        private static readonly ColumnConfiguration StartTimestampColumn = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{d458382b-1320-45c6-ba86-885da9dae71d}"), "StartTimestamp", "Android start timestamp for the event"),
```

Comment:
Remove Android. We don't know what kind of event this is including Chromium

Reply:
Fixed